### ISSUE TITLE
Feature: group layers but keep their position

### ISSUE THREAD

**Issue by jonnytest1** [mapeditor/tiled#3051]
i currently have in one map about 5 layers (above/below player *2 + event-layer) for one object

the idea is to flag all those layers as belonging to a certain tag and the ability to hide/grey out all the layers belonging to said tag (or maybe another veiw for all the tags with their layers)

to get some order into the continously growing layer list

**Comment by bjorn** [comment]
Hmm, so as far as I understand, due the desired rendering order, the group layers are not suitable for this purpose?

**Comment by jonnytest1** [comment]
yes group layers group from top to bottom - but my use case si to group in the horizontal axis if that makes sense ^^
